### DISCUSSION TITLE
Alternative a Doorbird

### DISCUSSION THREAD

**Discussion by EmAnzi3** [indomus/forum#843] [HUB: Home Assistant]
Salve,

Sto valutando l'idea di aggiornare il mio citofono, con un prodotto integrabile con HA. Ad oggi ho notato solo i Doorbird, con prezzi abbastanza elevati.

Conoscete altre soluzioni avente le stesse funzioni del Doorbird? Grazie.

**Comment by EmAnzi3** [comment]
Ci riprovo ;-)

**Reply by nromagna** [reply]
Hai dato un'occhiata a Hikvision. Con 300€ hai postazione esterna due interne e se hai altre telecamere della stessa marca comunicano tra loro. Io al momento ho due telecamere e funzionano benissimo con HA.
